.circleci/config.yml

```diff
@@ -1,44 +1,124 @@
----
-version: 2
-jobs:
-  build_and_test:
-    working_directory: ~/sfdx-plugin-lwc-test
+version: 2.1
+orbs:
+  release-management: salesforce/npm-release-management@4
+parameters:
+  run-auto-workflows:
```

Comment:
recommended: update this to the newer schedule pipelines version so we don't have to do it again
example: https://github.com/salesforcecli/plugin-signups/blob/49b769f9acfaf56263a2b185415cef7f9436d87e/.circleci/config.yml#L19